src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -508,6 +508,77 @@ def test_functionapp_consumption_linux_powershell(self, resource_group, storage_
         self.cmd('functionapp config show -g {} -n {}'.format(resource_group, functionapp_name), checks=[
             JMESPathCheck('linuxFxVersion', 'PowerShell|7.2')])
 
+class FunctionAppManagedEnvironment(LiveScenarioTest):
+    @ResourceGroupPreparer(location=WINDOWS_ASP_LOCATION_FUNCTIONAPP)
+    @StorageAccountPreparer()
+    def test_functionapp_create_with_appcontainer_managed_environment(self, resource_group, storage_account):
+        functionapp_name = self.create_random_name(
+            'functionappwindowsruntime', 40)
+        managed_environment_name = self.create_random_name(
+            'containerappmanagedenvironment', 40
+        )
+        plan_name = self.create_random_name(
+            'functionappplan', 40
+        )
+
+        self.cmd('containerapp env create --name {} --resource-group {} --location {}'
+        .format(managed_environment_name, resource_group, WINDOWS_ASP_LOCATION_FUNCTIONAPP)).assert_with_checks([
+                     JMESPathCheck('name', managed_environment_name),
+                     JMESPathCheck('resourceGroup', resource_group),
+                     JMESPathCheck('location', WINDOWS_ASP_LOCATION_FUNCTIONAPP)])
+
+        self.cmd('functionapp plan create -g {} -n {} --sku S1 --is-linux'.format(resource_group, plan_name))
+
+        self.cmd('functionapp create -g {} -n {} -p {} -s {} --environment {} --runtime dotnet --functions-version 4'
+                 .format(resource_group, functionapp_name, plan_name, storage_account, managed_environment_name)).assert_with_checks([
+                     JMESPathCheck('state', 'Running'),
+                     JMESPathCheck('name', functionapp_name),
+                     JMESPathCheck('kind', 'functionapp,linux'),
+                     JMESPathCheck('hostNames[0]', functionapp_name + '.azurewebsites.net')])
+
+        self.cmd('functionapp config show -g {} -n {}'.format(resource_group, functionapp_name), checks=[
+            JMESPathCheck('linuxFxVersion', 'DOCKER|mcr.microsoft.com/azuredocs/containerapps-helloworld:latest')])
```

Comment:
with a normal function app, do we expect a different image here?

Reply:
Yes, for a normal function app, this value would just be the image that the customer passed. But for function apps deployed on a managed environment, if the customer does not pass any image parameter, we want to use the same default hello world image used by container apps.

Reply:
so, it's already code in backend that this image will be the default image to be used right.

Reply:
I added the image to the _constants.py file